src/OrchardCore.Modules/OrchardCore.BackgroundTasks/Controllers/BackgroundTaskController.cs

```diff
@@ -43,8 +42,7 @@ public class BackgroundTaskController : Controller
             _backgroundTasks = backgroundTasks;
             _backgroundTaskManager = backgroundTaskManager;
             New = shapeFactory;
-            _siteService = siteService;
-            _notifier = notifier;
+            _siteService = siteService;            
```

Comment:
You could also remove from the constructor in this controller

Reply:
Resolved notifier object from constructor